### PR TITLE
Replace insensitive event types

### PR DESCRIPTION
**Pull request by mhagmajer**

### CODE REVIEW

File: Sources/Responses/EventType.swift [hunk 1]
```diff
@@ -68,9 +68,9 @@ public enum EventType: BoxEnum {
     /// 2 factor authentication enabled by user.
     case twoFactorEnabled
     /// Free user accepts invitation to become a managed user.
-    case masterInviteAccepted
```
Comment (mwwoda) [inline Sources/Responses/EventType.swift:71]: Is it not a breaking change if we are removing this case?

File: Sources/Responses/EventType.swift [hunk 2]
```diff
@@ -68,9 +68,9 @@ public enum EventType: BoxEnum {
...
+    case adminInviteAccepted
     /// Free user rejects invitation to become a managed user.
-    case masterInviteRejected
```
Comment (mwwoda) [inline Sources/Responses/EventType.swift:73]: Is it not a breaking change if we are removing this case?

File: Sources/Responses/EventType.swift (hunk 1 above)
Comment (jlawton) [reply Sources/Responses/EventType.swift:71]: It does seem breaking, but it's a rename, not a deletion, so the change is simple to accommodate. What's the implication of it being a breaking change? That we shouldn't make it? That we need to bump a major version?

**Comment by mhagmajer** [comment]
Thanks for the review @mwwoda. I've restored the removed cases.

**Review by mwwoda — DISMISSED** [review]
I'm approving it, but please check if it is possible to deprecate those old master* cases. I think we should discourage clients from using the old values.

**Pull request merged**